devicetypes/smartthings/testing/simulated-window-shade.src/simulated-window-shade.groovy

```diff
@@ -0,0 +1,248 @@
+/**
+ *  Copyright 2018, 2019 SmartThings
+ *
+ *  Provides a simulated window shade.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+
+metadata {
+	definition (name: "Simulated Window Shade", namespace: "smartthings", author: "SmartThings", runLocally: false, mnmn: "SmartThings", vid: "generic-window-shade") {
```

Comment:
namespace should be `smartthings/testing`

Reply:
Ooops! Thanks! Fixed